src/main/java/ch/njol/skript/registrations/Feature.java

```diff
@@ -11,20 +13,154 @@
+		"""
+			This feature includes:
+			
+			- The ability to reference a script in code.
+			- Finding and running functions by name.
+			- Reading configuration files and values.
+			""", "reflection",
+		LifeCycle.EXPERIMENTAL,
+		"[script] reflection"),
+	CATCH_ERRORS("Runtime Error Catching",
+		"""
+			A new catch [run[ ]time] error[s] section allows you to catch and suppress runtime errors within it and access them later with [the] last caught [run[ ]time] errors.
+			
+			```
+			catch runtime errors:
+			    ...
+			    set worldborder center of {_border} to {_my unsafe location}
+			    ...
+			if last caught runtime errors contains "Your location can't have a NaN value as one of its components":
+			    set worldborder center of {_border} to location(0, 0, 0)
+			```
+			""",
+		"catch runtime errors",
+		LifeCycle.EXPERIMENTAL,
+		"error catching [section]"),
+	TYPE_HINTS("Type Hints",
+		"""
+			Local variable type hints enable Skript to understand what kind of values your local variables will hold at parse time. Consider the following example:
+			
+			```
+			set {_a} to 5
+			set {_b} to "some string"
+			... do stuff ...
+			set {_c} to {_a} in lowercase # oops i used the wrong variable
+			```
+			
+			Previously, the code above would parse without issue. However, Skript now understands that when it is used, {_a} could only be a number (and not a text). Thus, the code above would now error with a message about mismatched types.
+			
+			Please note that this feature is currently only supported by simple local variables. A simple local variable is one whose name does not contain any expressions:
+			
+			```
+			{_var} # can use type hints
+			{_var::%player's name%} # can't use type hints
+			```
+			""",
+		"type hints",
+		LifeCycle.EXPERIMENTAL,
+		"[local variable] type hints"),
+	DAMAGE_SOURCE("Damage Sources",
+		"""
+			Damage sources are a more advanced and detailed version of damage causes. Damage sources include information such as the type of damage, the location where the damage originated from, the entity that directly caused the damage, and more.
+			
+			Below is an example of what damaging using custom damage sources looks like:
+			
+			```
+			damage all players by 5 using a custom damage source:
+				set the damage type to magic
+				set the causing entity to {_player}
+				set the direct entity to {_arrow}
+				set the damage location to location(0, 0, 10)
+			```
+			
+			For more details about the syntax, visit damage source on our documentation website.
+			""",
+		"damage source",
+		LifeCycle.EXPERIMENTAL,
+		"damage source[s]")
 	;
 
 	private final String codeName;
+	private final String displayName;
+	private final String description;
 	private final LifeCycle phase;
 	private final SkriptPattern compiledPattern;
 
-	Feature(String codeName, LifeCycle phase, String... patterns) {
+	Feature(@NotNull String displayName, @NotNull String description,
+			@NotNull String codeName, @NotNull LifeCycle phase,
```

Comment:
```suggestion
	Feature(@NotNull String displayName, @NotNull String codeName,
			@NotNull String description, @NotNull LifeCycle phase,
```
I think the names should be next to each other.